rules/aip0231/response_resource_field.go

```diff
@@ -31,7 +31,8 @@ var resourceField = &lint.MessageRule{
 		// Note: Use m.GetName()[8 : len(m.GetName())-9] to retrieve the resource
```

Comment:
```suggestion
		// Note: Use m.GetName()[8 : len(m.GetName())-8] to retrieve the resource
```

Comment doesn't match the implementation.

Reply:
No longer relevant.